packages/javascript-sdk/src/fr-auth/index.ts

```diff
@@ -108,16 +108,21 @@ abstract class FRAuth {
   ): Promise<FRStep | FRLoginSuccess | FRLoginFailure> {
     const parsedUrl = new URL(url);
```

Comment:
I'm starting to think we could really help with grabbing the `authIndexValue` and pass that into our tree config in the options object when we call next.